annotation/src/main/java/org/cbioportal/genome_nexus/annotation/web/AnnotationController.java

```diff
@@ -225,6 +225,32 @@ public AnnotationController(VariantAnnotationService variantAnnotationService,
         return isoformOverrideService.getIsoformOverrides(source);
     }
 
+    @ApiOperation(value = "getIsoformOverrideSources",
```

Comment:
@zheins Good point. Actually we have a documentation problem here. Current descriptions are parameter level only, there is no method level description. Since this method doesn't have any parameters, we ended up with no description.

I think we should add method level description for all endpoints. It might be better to address this issue with another PR, since we will need to modify all endpoints. According to the [swagger documentation](https://github.com/swagger-api/swagger-core/wiki/Annotations#apioperation), we need to update the `value` field of each `ApiOperation` to add a short description.